graylog2-server/src/main/java/org/graylog2/initializers/IndexerSetupService.java

```diff
@@ -114,11 +111,10 @@ protected void startUp() throws Exception {
         Tools.silenceUncaughtExceptionsInThisThread();
 
         LOG.debug("Starting indexer");
-        try {
-            node.start();
+        node.start();
```

Comment:
are we sure this and the next line never throw exceptions?
if they do, we would miss them and not print any error courtesy of `Tools.silenceUncaughtExceptionsInThisThread();`